Examples/Cxx/Circuits/SP_SynGen4OrderDQ_SMIB_Fault.cpp

```diff
@@ -0,0 +1,205 @@
+#include <DPsim.h>
```

Comment:
File should be renamed to `Examples/Cxx/Circuits/SP_SynGen4OrderDCIM_SMIB_Fault.cpp`